actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -410,18 +517,17 @@ def mask_token(raw_token) # :doc:
         encode_csrf_token(masked_token)
       end
 
-      def compare_with_real_token(token, session) # :doc:
-        ActiveSupport::SecurityUtils.fixed_length_secure_compare(token, real_csrf_token(session))
+      def compare_with_real_token(token) # :doc:
```

Comment:
```suggestion
      def compare_with_real_token(token, _session = nil) # :doc:
```

Since this method is `# :doc:` it means it's public API, so we might need to still accept the second parameter.